app/helpers/footer_helper.rb

```diff
@@ -0,0 +1,71 @@
+module FooterHelper
+  def footer_items(is_editor: false)
+    footer_items = []
+    if Flipflop.enabled?("design_system_edit_phase_3b".to_sym)
+      footer_items <<
+        {
+          title: "Reports, downtime and users",
```

Comment:
I wonder if this title should be `Reports and users` if we are not showing the "downtime" link?